packages/oauth/src/index.spec.js

```diff
@@ -417,8 +417,26 @@ describe('OAuth', async () => {
       assert.isTrue(sent);
     });
 
+    it('should call success callback when state query param is missing but stateValidation disabled', async () => {
+      const req = { url: 'http://someUrl.com?code=1234' };
```

Comment:
Let's use example.com for safety (I don't have any comments about the used domain but just in case)
```suggestion
      const req = { url: 'http://example.com?code=1234' };
```